tests/StorageApiTestCase.php

```diff
@@ -447,6 +447,32 @@ protected function createAndWaitForEvent(Event $event, Client $sapiClient = null
         }
     }
 
+    /**
+     * @return array
+     * @throws \Exception
+     */
+    protected function createWithFormDataAndWaitForEvent(Event $event, Client $sapiClient = null)
```

Comment:
Tady by asi bylo dobré to oddělit na "createEventSomehow" a "waitForEventId", ale už to nebudu komplikovat, jsou to jen testy a tohle je good enough